docs/concepts/sanitization.md

```diff
@@ -0,0 +1,55 @@
+React calls it dangerous for good reason -- if the HTML string has a `<script>` element or other javascript embedded into the dom elements,
+that javascript will be executed during deserialization. This is a security risk known as
+[cross site scripting (XSS)](https://en.wikipedia.org/wiki/Cross-site_scripting).
+
+## What this means for you
+
+### Sanitization
+Since Bandicoot does **NOT** automatically sanitize incoming or outgoing HTML, it is up to you as the developer/consumer to make sure that sanitization happens. However, Bandicoot does expose a [sanitizeHTML prop](/components/rich-text-editor.md#props) on the [RichTextEditor component](/components/rich-text-editor.md) which allows you to provide a sanitization function.
+
+If a sanitization function is provided through the `sanitizeHTML` prop, Bandicoot will pass incoming html strings through this function before inserting the content into the DOM (see [note 1](/concepts/sanitization.md#notes)). If provided, Bandicoot will also pass outgoing html through the `sanitizeHTML` function after serialization (see [note 2](/concepts/sanitization.md#notes)). If a `sanitizeHTML` function is not provided, Bandicoot will simply pass through the *unsanitized* html string - which is **NOT RECOMMENDED**!
+
+### The `sanitizeHTML` Function 
+When Bandicoot calls the `sanitizeHTML` function it passes two arguments:
+1. `suspectHTML`: An string of html content that potentially needs sanitizing.
+2. `actionType`: A string indicating what action Bandicoot is about to take.
+
+The `actionType` options and when they occur are as follows:
+- `setHTML`: Occurs right before inserting the `initialHTML` into the DOM as well as right before inserting content into the DOM through the `editorRef.current.setHTML()` and `editorRef.current.resetEditor()` functions.
+- `pasteHTML`: Occurs right before passing a paste operation's content to the [pasteFn prop](/components/rich-text-editor.md#props).  Note that the returned html string from the `pasteFn` function is immediately inserted into the DOM without further sanitization.
+- `insertContentEditableFalseHTML`: Occurs right before inserting content into the DOM using the `useContentEditableFalse` hook's `insertContentEditableFalseElement` function.
+- `getHTML`: Occurs post serialization, right before returning content through the `save()` or `editorRef.current.getHTML()` functions.
+- `initialSetLastSavedHTML`: Occurs right before initially setting the RichTextEditor's internal lastSavedHTML state variable.
+
+Bandicoot expects your `sanitizeHTML` function to return the sanitized html string.
+
+### Example
+```js
+<RichTextEditor
+  sanitizeHTML={myOwnSanitizeHtmlFunction}
+/>
+
+function myOwnSanitizeHtmlFunction(suspectHTML, actionType) {
+  // Implement your logic to sanitize 'suspectHTML' into 'sanitizedHTML'.
+  // If desired, key off of the 'actionType' string.
+  return sanitizedHTML;
+}
+```
```

Comment:
to @joeldenning's point, it'd be nice to have some real use cases for discriminating by `actionType`

Reply:
Add some additional points in the comments to `function myOwnSanitizeHtmlFunction...`